src/burn/engine/plan.cpp

```diff
@@ -853,6 +877,11 @@ static HRESULT ProcessPackage(
             hr = PlanLayoutPackage(pPlan, pPackage, wzLayoutDirectory);
             ExitOnFailure(hr, "Failed to plan layout package.");
         }
+        else if (BOOTSTRAPPER_ACTION_CACHE == pPlan->action)
```

Comment:
This doesn't seem right either. This will prevent a BA from deciding to do a _CACHE then set on of the packages as "Installed". _CACHE should set the default request state to cache and let the rest of the code flow through (in case the BA changes the default request state).,